src/FileProviders/Composite/src/Microsoft.Extensions.FileProviders.Composite.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RootNamespace>Microsoft.Extensions.FileProviders</RootNamespace>
+    <Description>Composite file and directory providers for Microsoft.Extensions.FileProviders.</Description>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.Extensions.FileProviders.Abstractions" />
```

Comment:
Out of curiosity, how does this work? Is this like a project or package reference from DNX?

Reply:
See https://github.com/aspnet/Extensions/blob/release/2.1/eng/targets/ResolveReferences.targets.

I've added a custom step to Reference items. The target resolves to either a ProjectRef or PackageRef dependency on a few factors, like (1) is this a servicing build (2) is this a test project (3) what was the version of used in the RTM release, etc.